Frontend/CO.CDP.OrganisationApp/Authorization/CustomScopeHandler.cs

```diff
@@ -67,4 +69,25 @@ protected override async Task HandleRequirementAsync(AuthorizationHandlerContext
 
         context.Fail();
     }
+
+    private Guid? GetOrganisationId()
```

Comment:
Temporarily copied over from UserInfoService which can't be injected.
Agreed to refactor on the next day with a comprehensive, global solution for accessing User.